CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 - Disallowed setting of `threads` in `profiles.yml` to a value greater than 1.
 
```

Comment:
This is not going to merge correctly. 0.21.8 has already been released, so it should be 0.21.9. However, 0.21.8 has been yanked temporarily due to an installation issue. I think it needs to be on hold until a fixed version of 0.21.8 is released.

Reply:
Ok, this confirms no security issues are present at the moment so we can hold off merging until everything's fixed in main.